core/src/main/kotlin/org/evomaster/core/output/service/naming/TestCaseNamingStrategy.kt

```diff
@@ -0,0 +1,13 @@
+package org.evomaster.core.output.service.naming
+
+import org.evomaster.core.output.NamingHelper
+import org.evomaster.core.output.TestCase
+import org.evomaster.core.search.EvaluatedIndividual
+
+interface TestCaseNamingStrategy {
```

Comment:
when declaring interfaces, then all methods need some JavaDocs. 
also, if these methods require solution to be pass in constructor, and not as input parameter, then maybe to consider to have this class rather as abstract, with solution saved in a protected field

Reply:
Will change to an abstract class then, makes more sense